### DISCUSSION TITLE
Rendering SLS 'base:firewall' failed: Jinja variable 'list object' has no attribute 'values'

### DISCUSSION THREAD

**Discussion by squares-ui** [Security-Onion-Solutions/securityonion#2044] [Unsupported Versions] [answered]
FYI 
Clean install
esxi snapshot
restored (due to breaking  issues)
running so-allow with priv

![image](https://user-images.githubusercontent.com/21036586/99956943-cc33ad00-2d7e-11eb-92ce-58eeb99017b8.png)

Followed other thread, restarted salt-minion, and tried again

![image](https://user-images.githubusercontent.com/21036586/99956990-dfdf1380-2d7e-11eb-98fd-e5a1377ed305.png)

But the same happened again.

**Comment by m0duspwnens** [comment]
@squares-ui 
Which type of install is this? Distributed, standalone, etc. Were you running so-allow above on the manager node?

**Comment by squares-ui** [comment, marked as answer]
Standalone, simplest possible.
There is a chance the snapshot was taken on a DHCP (before I realised how much issues that caused), so rebuilding again lol

Apologies I did try and come to delete this but couldn't see how

**Reply by m0duspwnens** [reply]
Sounds good, let us know if the issue persists after the rebuild.

I did open a potential issue (https://github.com/Security-Onion-Solutions/securityonion/issues/2051) related to the screenshot. The issue is that so-allow may not be able to apply the firewall state if another state.apply is running is the background preventing the rules from being applied right after so-allow. The rules would eventually be applied when the next highstate is run, which is every 15 minutes by default.
